client/layout/guided-tours/test/tour-branching.js

```diff
@@ -0,0 +1,115 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { tourBranching } from '../tour-branching';
+
+describe( 'Guided Tours Branching', () => {
+	test( 'tourBranching returns a branch tree for a tour JSX tree', () => {
+		// We are testing just a tree of elements created by React.createElement. The actual
+		// component types don't matter (never rendered) and can be null.
+		const Fragment = 'Fragment';
+		const Tour = 'Tour';
+		const Step = 'Step';
+		const ButtonRow = 'ButtonRow';
+		const SiteLink = 'SiteLink';
+
+		// These components have `step` props and are checked for `displayName` and `name`
+		const Continue = () => null;
+		const Next = () => null;
+
+		const tourTree = (
+			<Tour name="checklistPublishPost" version="20171205">
+				<Step name="init">
+					{ ( { translate } ) => (
+						<Fragment>
+							<p>{ translate( 'It’s time to get your blog rolling with your first post.' ) }</p>
+							<ButtonRow>
+								<Next step="categories-tags">{ translate( 'All done, continue' ) }</Next>
+								<SiteLink>{ translate( 'Return to the checklist' ) } </SiteLink>
+								<Continue step="categories-tags" hidden />
+							</ButtonRow>
+						</Fragment>
+					) }
+				</Step>
+
+				<Step name="categories-tags">
+					{ ( { translate } ) => (
+						<Fragment>
+							<p>{ translate( 'Categories and Tags' ) }</p>
```

Comment:
Hi! I've found a possible matching string that has already been translated **92** times:
`translate( 'Categories and Tags Converter' )` *ES Score: 11*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/guided-tours-branching)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).